sdk/communication/Azure.Communication.JobRouter/samples/Sample2_ClassificationWithQueueSelector.md

```diff
@@ -0,0 +1,306 @@
+# Azure.Communication.JobRouter Samples - Using QueueSelectors in Classification Policy
```

Comment:
I preview 2, we've simplified the queue selector to only be a label selector, so I'm not sure if it's worth documenting the others.

Reply:
removed QueueIdSelector